src/app/app-routing.module.ts

```diff
@@ -1,10 +1,25 @@
-import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
+import {NgModule} from '@angular/core';
+import {RouterModule, Routes} from '@angular/router';
 
-const routes: Routes = [];
+import {AddMovieComponent} from "./components/add-movie/add-movie.component";
+import {WrapperComponent} from "./components/wrapper/wrapper.component";
+
+const routes: Routes = [
+  {
+    path: '',
+    component: WrapperComponent,
+    pathMatch: 'full'
+  },
+  {
+    path: 'add',
+    component: AddMovieComponent,
```

Comment:
it could be as AddMoviewPageComponent, so clearly we'll see that this is a page component that serves as an entry point and holds other components